### ISSUE TITLE
Multicall is supports writing function calls?

### ISSUE THREAD

**Issue by fqlx** [makerdao/multicall#40]
In the multicall.sol contract: https://github.com/makerdao/multicall/blob/master/src/Multicall.sol#L7
```
Aggregate results from multiple read-only function calls
```

As far as I'm aware, it supports writing to function calls.  Is this correct?
And it does not support `payable` or passing `msg.value`?

**Comment by brian-man** [comment]
Agreed.  There needs to be `multistaticcall()` with `external view`.  Right?
